core/templates/dev/head/pages/exploration_editor/lost_changes_modal.html

```diff
@@ -0,0 +1,22 @@
+<div class="modal-header">
+  <h3>Error With Exploration</h3>
+</div>
+
+<div class="modal-body">
+  <p>
+    Sorry! The following changes will be lost. This is likely due to
+    editing an exploration without internet connection.
```

Comment:
Can we be a bit more explicit? What exactly happened? (Was their draft overwritten, for example?)

You don't want to give the impression that editing an exploration without Internet connection messes things up automatically, if that's not accurate.